lib/OpenCloud/LoadBalancer/Service.php

```diff
@@ -51,11 +52,20 @@ public function loadBalancer($id = null)
      */
     public function loadBalancerList($detail = true, array $filter = array())
     {
+        $options = $this->makeResourceIteratorOptions($this->resolveResourceClass('LoadBalancer'));
+
+        if (isset($filter['limit'])) {
+            $options['limit.page'] = $filter['limit'];
```

Comment:
> either way the query is set and the URL will have the appropriate value.

How will the URL have the appropriate value without you explicitly setting `$options['limit.page']`? Is it part of the paginated "next page" URL returned in the API response?